website/content/docs/builders/outscale/index.mdx

```diff
@@ -105,13 +105,13 @@ export OUTSCALE_X509KEY="the/path/to/your/x509key"
 ### Checking that system time is current
 
 Outscale uses the current time as part of the [request signing
-process](http://docs.aws.osc.com/general/latest/gr/sigv4_signing.html). If
+process](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html). If
```

Comment:
does outscale not have their own version of these docs? I know their API directly and intentionally mirrored AWS for a while but it surprises me they didn't also mirror the docs

Reply:
Ah good catch. I somehow thought this was pointing outscale. I found this URL https://wiki.outscale.net/display/EN/About+Signatures+of+API+Requests which is probably where this link should point to now.